test/e2e/performanceprofile/functests/5_latency_testing/latency_testing.go

```diff
@@ -224,8 +224,8 @@ func getValidValuesTests(toolToTest string) []latencyTest {
 
 	successRuntime := "30"
 	// Using a timeout value such that ginkgo timeout > runtime + latency to ensure successfull runs
-	successGinkgoTimeout := "400s"
-	testSet = append(testSet, latencyTest{testDelay: "200", testRuntime: successRuntime, testMaxLatency: untunedLatencyThreshold, testCpus: "4", outputMsgs: []string{success}, toolToTest: toolToTest, ginkgoTimeout: successGinkgoTimeout})
+	successGinkgoTimeout := "200s"
+	testSet = append(testSet, latencyTest{testDelay: "20", testRuntime: successRuntime, testMaxLatency: untunedLatencyThreshold, testCpus: "4", outputMsgs: []string{success}, toolToTest: toolToTest, ginkgoTimeout: successGinkgoTimeout})
```

Comment:
I believe we should keep one test with 200 seconds of delay, because we had a bug earlier that failed to run the tests with more than 120s seconds:
https://issues.redhat.com/browse/OCPBUGS-5219 
maybe you can reduce it a bit but better to keep it above 120s.